### DISCUSSION TITLE
Email Verification error

### DISCUSSION THREAD

**Discussion by Mmwangi21** [community/community#116378] [GitHub Education]
### Hi everyone,

I am currently applying for Github student developer pack using my student email. The email remains unverified , I haven't received the verification message
username-Mmwangi21 ,student email-sc450421@students.mu.ac.ke
error experiencing
sc450421@student.mu.ac.ke
Unverified

**Comment by mishmanners** [comment]
Usually for a student email you need a .edu account. There is also a place where you can upload your student ID as verification. Have you done that?

**Comment by samus-aran** [comment]
Hi @Mmwangi21 👋 

You can reach out to support and ask for assistance on verifying your email address to then proceed with the student verification. We can confirmed that other users have successfully verified this domain - try checking your spam folders.
